lib/shared/bucketing-assembly-script/__tests__/bucketing/bucketing.test.ts

```diff
@@ -1309,7 +1446,7 @@ describe('Client Data', () => {
         const c1 = generateBucketedConfig(user)
         expect(c1).toEqual(
             expect.objectContaining({
-                featureVariationMap: {},
+                featureVariationMap: {'614ef6aa473928459060721a': '6153553b8cf4e45e0464268d'},
```

Comment:
Why are these added here now?

Reply:
I mentioned this in the description briefly, but its because I added the all users target to the test data